src/commands/createFunctionApp/FunctionAppCreateStep.ts

```diff
@@ -39,10 +39,6 @@ export class FunctionAppCreateStep extends AzureWizardExecuteStep<IFunctionAppWi
         context.telemetry.properties.newSiteMinorVersion = stack.minorVersion.value;
         context.telemetry.properties.planSkuTier = context.plan?.sku?.tier;
 
-        const message: string = localize('creatingNewApp', 'Creating new function app "{0}"...', context.newSiteName);
-        ext.outputChannel.appendLog(message);
-        progress.report({ message });
```

Comment:
I would add the `progress.report()` back in because we still need to emit progress events